kratos/python_scripts/python_solver.py

```diff
@@ -187,6 +187,29 @@ def _ImportModelPart(self, model_part, model_part_import_settings):
                 KratosMultiphysics.ReorderAndOptimizeModelPartProcess(model_part, tmp).Execute()
             KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished reading model part from mdpa file.")
 
+        elif (input_type == "cad.json"):
+            problem_path = os.getcwd()
+            input_filename = model_part_import_settings["input_filename"].GetString()
+
+            # Import model part from mdpa file.
+            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Reading CAD model from file: " + os.path.join(problem_path, input_filename) + ".json")
+            KratosMultiphysics.ModelPartIO(input_filename).ReadModelPart(model_part)
+            KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished reading CAD model.")
+
+            # Create integration domain and elements.
+            if model_part_import_settings.Has("physics_input_type"):
+                physics_input_type = model_part_import_settings["physics_input_type"].GetString()
+                if (physics_input_type == "ph.json"):
```

Comment:
```suggestion
                if physics_input_type == "ph.json":
```
More pythonic

Reply:
If you want. I'm totally fine with that ;)